src/logged-evaluation.nix

```diff
@@ -38,13 +36,13 @@ let
     name = "lorri-keep-env-hack-${drv.name}";
 
     origBuilder = drv.builder;
-    builder = builtins.storePath nixConfig.shell;
+    builder = "/bin/sh";
```

Comment:
```
# We can do this because nix treats the string `"/bin/sh"` specifically
# and mounts the path w/ closure into its sandbox. It’s theoretically possible
# that a user compiles nix without that path, but we don’t support it.
```

Reply:
Nice. Can you add this as a suggestion, and I'll just click "Accept" and then merge? :) (also, `nix` -> `Nix`)